pyg_lib/csrc/sampler/cpu/utils.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <torch/torch.h>
+
+template <typename scalar_t>
+inline scalar_t randint(scalar_t low, scalar_t high) {
+  const auto dtype = c10::CppTypeToScalarType<scalar_t>::value;
+  const auto options = torch::TensorOptions().dtype(dtype);
+  const auto rand = torch::randint(low, high, {1}, options);
+  return rand.template data_ptr<scalar_t>()[0];
```

Comment:
It is related to torch's random seed. An STL random is more efficient but affects reproducibility I think

Reply:
We could look into how `torch::randint` is implemented internally.

Reply:
For performance, we could refactor our random module later so that a single `randint` will not cause too much overheads. I could do that in another PR.

Reply:
Super :)

Reply:
As long as we add a todo. I'm ok.